src/components/Select/Select.jsx

```diff
@@ -31,9 +31,9 @@ class Select extends React.Component {
       errorSpanId = `${this.selectId}-error-message`;
       errorSpan = (
         <span
-          className="usa-input-error-message"
+          className='usa-input-error-message'
```

Comment:
I had prettier on by default, I _undid_ a lot of the changes made on save (you can see in commmits). There's a few left over like this.

Reply:
Yeah, we still need to add an `.eslintrc` to this repo. :sweat_smile: